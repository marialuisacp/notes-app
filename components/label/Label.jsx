import React from 'react';
import PropTypes from 'prop-types';
import Styledlabel from './styles/Label.styles';
import { colors } from './defaultLabelColors';

const Label = ({ color }) => (
  <Styledlabel color={(color && colors && colors[color] || 'red')}></Styledlabel>
);

Label.propTypes = {
  color: PropTypes.string,
};

export default Label;