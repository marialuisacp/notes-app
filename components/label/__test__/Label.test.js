import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Label from '../Label';

const defaultProps = {
  color: 'red'
};

test('should render component Label', () => {
  const { fragment } = render(<Label {...defaultProps}></Label>);
  expect(fragment).toMatchSnapshot();
});
