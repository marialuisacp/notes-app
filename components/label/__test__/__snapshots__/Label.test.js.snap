// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Label 1`] = `undefined`;
